app/models/follow.rb

```diff
@@ -0,0 +1,15 @@
+class Follow < ApplicationRecord
+    belongs_to :follower, class_name: 'User'
+    belongs_to :followee, class_name: 'User'
```

Comment:
You need to rename your database columns to `follower_id` and `followee_id` or specify the foreign_key so rails can know what column to look for